Lesson8/client.py

```diff
@@ -0,0 +1,205 @@
+import sys
+import json
+import socket
+import time
+import argparse
+import logging
+import threading
+import logs.config_client_log
+from common.variables import DEFAULT_PORT, DEFAULT_IP_ADDRESS, ACTION, \
+    TIME, USER, ACCOUNT_NAME, SENDER, PRESENCE, RESPONSE, \
+    ERROR, MESSAGE, MESSAGE_TEXT, DESTINATION, EXIT
+from common.utils import get_message, send_message
+from errors import IncorrectDataReceivedError, ReqFieldMissingError, \
+                   ServerError
+from decos import log
+
+
+LOGGER = logging.getLogger('client')
+
+
+@log
+def create_exit_message(account_name):
+    return {
+        ACTION: EXIT,
+        TIME: time.time(),
+        ACCOUNT_NAME: account_name
+    }
+
+
+@log
+def message_from_server(sock, my_username):
```

Comment:
Client seems broken as there's no more menu:
![image](https://user-images.githubusercontent.com/2947151/89711996-979a1200-d996-11ea-93f0-ce9701025331.png)
![image](https://user-images.githubusercontent.com/2947151/89712007-aaace200-d996-11ea-871d-0b4358f8731b.png)

Reply:
I fixed it